Classes/Utility/String+BinaryFile.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  String+BinaryFile.swift
+//  Freetime
+//
+//  Created by Ivan Magda on 01/03/2018.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+// MARK: String+BinaryFile
+
+extension String {
```

Comment:
Do we really want to namespace all of this under `String`?

Reply:
How about `FilePath` ?

Reply:
👍